godel_process_execution/src/abb_blend_process_service.cpp

```diff
@@ -207,17 +207,19 @@ bool godel_process_execution::AbbBlendProcessService::executeProcess(
     return false;
   }
 
-  if (!req.wait_for_execution)
+  if (req.wait_for_execution)
```

Comment:
I assume that these changes are an artifact of your work on tracking the progress of execution?